### PR TITLE
Add cpuid-related sources for other architectures to FIPS sources

### PR DESCRIPTION
**Pull request by t8m**
They need to be explicitly added otherwise only x86_64 arch ones will be there which is incorrect.

The second commit applies only to master branch. The first also to 3.x branches.

### CODE REVIEW

**Review by paulidale — APPROVED** [review]

**Comment by t8m** [comment]
ping @openssl/committers for second review

File: Configurations/unix-Makefile.tmpl [hunk 1]
```diff
@@ -1260,7 +1260,8 @@ providers/fips.module.sources.new: configdata.pm
 		   crypto/ec/asm/*.pl \
 		   crypto/modes/asm/*.pl \
 		   crypto/sha/asm/*.pl \
-		   crypto/x86_64cpuid.pl; do \
+		   crypto/*cpuid.pl crypto/*cpuid.S \
+		   crypto/*cap.c; do \
```
Comment (tmshort) [inline Configurations/unix-Makefile.tmpl:1264]: Nit: I suggest the files be on their own line, but there is prior precedence for having two files per line.

**Review by beldmit — APPROVED** [review]
LGTM. aes-gcm-siv is removed intentionally, right?

File: providers/fips.module.sources
Comment (tmshort) [inline providers/fips.module.sources:1]: A bit disappointed in the number of files that are now added to the FIPS checksum. Do the cpuid files really need to be added? I guess they don't change that much?

**Review by tmshort — APPROVED** [review]
Two minor comments, nothing to block approval.

File: providers/fips.module.sources
Comment (t8m) [reply providers/fips.module.sources:1]: They are part of the fips module sources, that means they need to be added there. Also I do not see the reason for disappointment.

**Comment by tmshort** [comment]
> > LGTM. aes-gcm-siv is removed intentionally, right?
> 
> Yes, it is not FIPS-approved and included in FIPS module.

Yeah, add in SIV and everything breaks.

**Pull request closed without merging**